### DISCUSSION TITLE
Aplicativos competidores

### DISCUSSION THREAD

**Discussion by BeyondMagic** [cerejeiros/mobile#8] [Show and tell]
+ [TickTick](https://ticktick.com/about/features) 9/10
+ Android, Web

Possuí tudo que um aplicativo como o nosso precisaria, tendo a única desvantagem que algumas coisas precisam ser pagas mensalmente ($10).

+ [clickup](https://clickup.com/features) 7/10
+ Web

Não possuí blocos de rotina e aplicativo, mas é muito simples e intuitivo de se usar sem precisar de *how-to*s. O design é uma arte por definição, sendo o maior problema que a versão de graça é muito limitada.

@cerejeiros

**Comment by Neoprot** [comment]
+ [Productive] (https://productiveapp.io) 9/10
+ Android, IOS

Possui tudo que um aplicativo precisaria tem bastante sensação de progresso e visual, porém acho que o excesso de informações na criação de hábitos pra um desavisado poderia assustar, não consegui fazer configuração para lembretes com notificações no celular, tem uma aba de desafios muito interessante (um exemplo de gamificação), tem uma aba de pagamento mensal, porém não fui impedido de nada por não assinar, e também n achei restrições.

@cerejeiros
